src/Umbraco.Web.UI.Client/src/less/forms.less

```diff
@@ -653,28 +654,21 @@ input.search-query {
   padding-left: 14px;
   padding-left: 4px \9; /* IE7-8 doesn't have border-radius, so don't indent the padding */
   margin: 0; // Remove the default margin on all inputs
-  .border-radius(0px);
 }
 
 /* Allow for input prepend/append in search forms */
-.form-search .input-append .search-query,
-.form-search .input-prepend .search-query {
-  .border-radius(0); // Override due to specificity
-}
-.form-search .input-append .search-query {
-  .border-radius(14px 0 0 14px);
-}
-.form-search .input-append .btn {
-  .border-radius(0 14px 14px 0);
-}
-.form-search .input-prepend .search-query {
-  .border-radius(0 14px 14px 0);
+.form-search {
+    .input-prepend {
+        .btn {
+            .border-radius(0 @borderRadiusSmall @borderRadiusSmall 0);
+        }
+    }
+    .input-append {
+        .btn {
+            .border-radius(0 @borderRadiusSmall @borderRadiusSmall 0);
+        }
+    } 
```

Comment:
This is simplified since we only need to change border-radius on `input-prepend` and `input-append`.